src/main/java/uk/gov/hmcts/reform/userprofileapi/util/UserProfileValidator.java

```diff
@@ -99,11 +99,21 @@ static boolean validateAndReturnBooleanForParam(String param) {
     }
 
     static void validateUserIds(UserProfileDataRequest userProfileDataRequest) {
-        if (userProfileDataRequest.getUserIds().isEmpty()) {
+        if (CollectionUtils.isEmpty(userProfileDataRequest.getUserIds())
+             || userProfileDataRequest.getUserIds().contains(" ")
+             || userProfileDataRequest.getUserIds().stream().anyMatch(userId -> isUserIdInValid(userId))) {
             throw new RequiredFieldMissingException("no user id in request");
         }
     }
 
+    static boolean isUserIdInValid(String userId) {
+        boolean isInValid = false;
+        if (StringUtils.isBlank(userId)) {
+            isInValid = true;

```

Comment:
don't require this method , you can directly use anyMatch(userId -> StringUtils.isBlank(userId))

Reply:
removed the code